configserver/src/main/java/com/yahoo/vespa/config/server/ApplicationRepository.java

```diff
@@ -799,26 +808,20 @@ private int servicePort(ServiceInfo serviceInfo) {
         application.getModel().getHosts().stream()
                 .filter(host -> host.getServices().stream().noneMatch(serviceInfo -> serviceInfo.getServiceType().equalsIgnoreCase("logserver")))
```

Comment:
I know this wasn't changed now, but we should probably extract this filter predicate into a separate method with a semantic name.